OpenXmlPowerTools/PegBase.cs

```diff
@@ -1569,69 +2011,100 @@ internal Trie(char cThis,int nIndex, string[] literals)
                 internal Trie[] children_;      //contains the successor node of cThis_;
             }
+
             internal Trie literalsRoot;
+
             public OptimizedLiterals(string[] litAlternatives)
             {
                 literalsRoot = new Trie('\u0000', 0, litAlternatives);
             }
         }
-        #endregion  PEG optimizations
+
+        #endregion PEG optimizations
+
         #region Constructors
-        public PegCharParser():this("")
-        {
-           
 
+        public PegCharParser() : this("")
+        {
         }
-        public PegCharParser(string src):base(null)
+
+        public PegCharParser(string src) : base(null)
         {
             SetSource(src);
         }
-        public PegCharParser(string src, TextWriter errOut):base(errOut)
+
+        public PegCharParser(string src, TextWriter errOut) : base(errOut)
         {
             SetSource(src);
             nodeCreator_ = DefaultNodeCreator;
         }
+
         #endregion Constructors
+
         #region Overrides
+
         public override string TreeNodeToString(PegNode node)
         {
-            string label = base.TreeNodeToString(node);
+            var label = base.TreeNodeToString(node);
             if (node.id_ == (int)ESpecialNodes.eAnonymousNode)
             {
-                string value = node.GetAsString(src_);
-                if (value.Length < 32) label += " <" + value + ">";
-                else label += " <" + value.Substring(0, 29) + "...>";
+                var value = node.GetAsString(src_);
+                if (value.Length < 32)
+                {
+                    label += " <" + value + ">";
+                }
+                else
+                {
+                    label += " <" + value.Substring(0, 29) + "...>";
+                }
             }
             return label;
         }
+
         #endregion Overrides
+
         #region Reinitialization, Source Code access, TextWriter access,Tree Access
+
         public void Construct(string src, TextWriter Fout)
         {
             base.Construct(Fout);
             SetSource(src);
         }
+
         public void SetSource(string src)
         {
-            if (src == null) src = "";
+            if (src == null)
+            {
+                src = "";
+            }
+
             src_ = src; srcLen_ = src.Length; pos_ = 0;
-            errors.lineStarts = new SortedList<int, int>();
-            errors.lineStarts[0] = 1;
+            errors.lineStarts = new SortedList<int, int>
+            {
+                [0] = 1
+            };
+        }
+
+        public string GetSource()
+        {
+            return src_;
         }
-        public string GetSource() { return src_; }
+
         #endregion Reinitialization, Source Code access, TextWriter access,Tree Access
+
         #region Setting host variables
-        public bool Into(Matcher toMatch,out string into)
+
+        public bool Into(Matcher toMatch, out string into)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename the parameter 'into' so that it does not duplicate the method name.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)